### PR TITLE
Feature/order samples unsequa

### PR DESCRIPTION
**Pull request by chahank**
Changes proposed in this PR:
- Add two examples to the tutorials on how to use the unseques modules for coupled input variables and for handling efficiently the loading of multiple large files
- Adds a method to order the samples in unsequa

~~Note: this pull requests uses the loglevel from #765 . All changes not in the tutorial file are due to the other pull requests. Please merge #765 first.~~

### PR Author Checklist

- [ ] Read the [Contribution Guide][contrib]
- [ ] Correct target branch selected (if unsure, select `develop`)
- [ ] Descriptive pull request title added
- [ ] Source branch up-to-date with target branch
- [ ] [Documentation](https://climada-python.readthedocs.io/en/latest/guide/Guide_PythonDos-n-Donts.html#2.--Commenting-&-Documenting) updated
- [ ] [Tests][testing] updated
- [ ] [Tests][testing] passing
- [ ] No new [linter issues][linter]
- [ ] [Changelog](https://github.com/CLIMADA-project/climada_python/blob/main/CHANGELOG.md) updated

### PR Reviewer Checklist

- [ ] Read the [Contribution Guide][contrib]
- [ ] [CLIMADA Reviewer Checklist](https://climada-python.readthedocs.io/en/latest/guide/Guide_Reviewer_Checklist.html) passed
- [ ] [Tests][testing] passing
- [ ] No new [linter issues][linter]

[contrib]: https://github.com/CLIMADA-project/climada_python/blob/main/CONTRIBUTING.md
[testing]: https://climada-python.readthedocs.io/en/latest/guide/Guide_Continuous_Integration_and_Testing.html
[linter]: https://climada-python.readthedocs.io/en/stable/guide/Guide_Continuous_Integration_and_Testing.html#3.C.--Static-Code-Analysis

### CODE REVIEW

**Comment by chahank** [comment]
Note: tests for petals fail due to the landslide module. This is unrelated to this PR....

**Comment by timschmi95** [comment]
@chahank Where in the notebook are the added examples? The table of contents is the same between the two versions.
If I know which examples, I can review this added part although I've never used the unsequa module. Someone who has used it may be able to provide better feedback regarding how the added parts tie in with the unseque module in general.

**Comment by chahank** [comment]
It is all the "advanced examples" at the bottom. I do not know why the sections do not show up in the t.o.c., I will have to fix this. Thank!

File: climada/engine/unsequa/unc_output.py [hunk 1]
```diff
@@ -112,6 +112,22 @@ def __init__(self, samples_df, unit=None):
         self.samples_df = samples_df
         self.unit = unit
 
+    def order_samples(self, by):
```
Comment (timschmi95) [inline climada/engine/unsequa/unc_output.py:115]: The docstring has the argument name 'by_unc_params' instead of 'by'.

**Review by timschmi95 — COMMENTED** [review]
Without having used the unsequa module I think the the point comes across how one can (1) share an input parameter among multiple input variables and (2) using a global variable to avoid reloading files. 
But a second reviewers opinion, who has used the unsequa module, is needed as I didn't follow the details in the code. 
Regarding layout, if there is an option the not print the details (e.g. Cost Benefit table) for each variable combination it would improve the readability of the tutorial. 
Lastly, would pulling the latest changes from the target branch `feature/unsequa_multiprocessing` fix the failing tests as they seem to pass in PR #763 ?

**Pull request merged**